app/controllers/steps_controller.rb

```diff
@@ -14,8 +14,14 @@ def update
     return render plain: status_mismatch_error(step), status: :conflict if params['current-status'] && step.status != params['current-status']
 
     step.update(parser.to_h)
+
+    # TODO: Do we only want to do this for certain updates, e.g., when setting status to completed?
+    # Enqueue next steps
+    next_steps = NextStepService.for(step: step)
+    next_steps.each { |next_step| Queue.enqueue(next_step) }
```

Comment:
Are there some steps that we don't want to enqueue?

Reply:
Yes, if the template has skipQueue as an attribute I don't think we want to enqueue it.  I think NextStepService is only returning things that can be queued though: https://github.com/sul-dlss/workflow-server-rails/blob/master/app/services/next_step_service.rb#L28